templates/.phpcs.xml.dist

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
```

Comment:
This naming is a bit misleading - perhaps `"WordPress Coding Standards for this plugin"? is better, and less implies that there's a WPCS specifically for Plugins?

Reply:
I'm pulling this name from the current version, but I am good to do this naming change.

Reply:
May I suggest "_WPCS based custom ruleset for plugins_" or "_WordPress Coding Standards based custom ruleset for your plugin/for plugin xyz_" ?

Makes it very clear that this is a _custom ruleset_ and not a _standard_. There is a significant difference between those two and naming this right will help people find the right information if they would search for more info.

To explain the difference:
* A **standard** can have add new sniffs, can be installed and referenced by name.
* A **custom ruleset** uses existing standards and ruleset and may add configuration for these.